.delivery/build-cookbook/recipes/deploy.rb

```diff
@@ -4,3 +4,57 @@
 #
 # Copyright (c) 2015 The Authors, All Rights Reserved.
 include_recipe 'delivery-truck::deploy'
+
+#
+# We publish to supermarket.chef.io when a change hits delivered
+#
+# TODO: The following code needs to be incorporated into `delivery-truck`
+#       in some manner.
+#
+if delivery_environment == 'delivered'
+  require 'uri'
+
+  supermarket_creds = encrypted_data_bag_item_for_environment('creds', 'supermarket')
```

Comment:
So technically you don't need it since there is a transitive dependency on **chef-sugar** through **delivery-truck**. That being said, it's good style to properly declare the **chef-sugar** dep so I updated the `metadata.rb`.